### PR TITLE
Rate limit domain cache refresh

### PR DESCRIPTION
**Pull request by yycptt**
**What changed?**
Skip domain cache refresh if the time duration is too short since last refresh

**Why?**
If a domain doesn't exist and client keeping calling API for that domain, domain cache will be refreshed for every call, causing a large load on persistence layer and other APIs to fail due to service busy error.

**How did you test it?**
Unit test

**Potential risks**
N/A

### CODE REVIEW

**Comment by yycptt** [comment]
@wxing1292

File: common/cache/domainCache.go [hunk 1]
```diff
@@ -476,6 +483,9 @@ UpdateLoop:
 		}
 	}
 
+	// only update last refresh time when refresh succeeded
+	c.lastRefreshTime = now
```
Comment (wxing1292) [inline common/cache/domainCache.go:487]: could you put this at the very end?

**Comment by wxing1292** [comment]
LGTM

File: common/cache/domainCache.go (hunk 1 above)
Comment (yycptt) [reply common/cache/domainCache.go]: Done.

**Pull request merged**